### PR TITLE
doc: add minutes for meeting oct 11 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-10-11.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+### Workgroup Update
+
+#### Release Work Group
+
+* Major work, formal chartering of Release WG, Michael moved forward,
+  some cleanup still do to.
+* v6.x release upcoming.
+* Preparing for V8.x
+* Gibson starting to take over LTS release, Gibson will be mentoring
```
Comment (sam-github) [inline meetings/2017-10-11.md]: "myles will be mentoring" I think

File: meetings/2017-10-11.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+* Another focus is working towards sharing the backporting work so
+  that more people can do it.
+* Last discussion was that it.s likely too late for 6.2 in 8.x before
```
Comment (sam-github) [inline meetings/2017-10-11.md]: "it's"

File: meetings/2017-10-11.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  * at the summit, groups split out and filled in stickies, and the
+    Grouped on the wall.
+  * General hope was to do more in different areas (ex Australia,
+    China).
+  * Myles may try to run at NodeFest and over next few months
+  * Those who led, Sarah (new board member, Tracy etc will be pushing
+    forward).
+  * Assumption is that is is going to be a multi month effort.
+  * Michael: suggest that although it refers to Values, we likely
+    won.t change the wording significantly based on what Values are
```
Comment (fhinkel) [inline meetings/2017-10-11.md]: won't

File: meetings/2017-10-11.md (hunk 1 above)
Comment (mhdawson) [reply meetings/2017-10-11.md]: fixed

File: meetings/2017-10-11.md (hunk 2 above)
Comment (mhdawson) [reply meetings/2017-10-11.md]: fixed

File: meetings/2017-10-11.md (hunk 3 above)
Comment (mhdawson) [reply meetings/2017-10-11.md]: fixed

**Comment by mhdawson** [comment]
Landed as 6a408f90db2653e942df5d1986b112951bccd550

**Pull request closed without merging**